Tests/TuistCacheTests/ContentHashing/SourceFilesContentHasherTests.swift

```diff
@@ -0,0 +1,64 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+final class SourceFilesContentHasherTests: TuistUnitTestCase {
+    private var subject: SourceFilesContentHasher!
+    private var mockContentHasher: MockContentHashing!
+    private var temporaryDirectory: TemporaryDirectory!
```

Comment:
You can use [this method](https://github.com/tuist/tuist/blob/master/Sources/TuistSupportTesting/TestCase/TuistTestCase.swift#L64) from each test function.

Reply:
👍 done